content/en/blog/_posts/2025-xx-xx-jobs-managedby-goes-ga.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog
+title: "Kubernetes v1.35: Job Managed By Goes GA"
+date: 2025-XX-XX
+draft: true
+slug: kubernetes-v1-35-job-managedby-for-jobs-goes-ga
+author: >
+  [Dejan Zele Pejchev](https://github.com/dejanzele) (G-Research),
+  [Michał Woźniak](https://github.com/mimowo) (Google)
+---
+
+In Kubernetes v1.35, the managedBy field for Jobs graduated to General Availability (GA).
+
+This feature allows external controllers to take full responsibility for Job reconciliation, unlocking powerful scheduling patterns like multi-cluster dispatching with [MultiKueue](https://kueue.sigs.k8s.io/docs/concepts/multikueue/).
+
+## Why delegate Job reconciliation?
+
+The primary motivation for this feature is to support multi-cluster batch scheduling architectures, such as MultiKueue.
+
+Specifically in the MultiKueue design, we distinguish between a Management Cluster and a pool of Worker Clusters:
```

Comment:
```suggestion
Specifically in the MultiKueue design, the architecture distinguish between a Management Cluster and a pool of Worker Clusters:
```
Should we update this to avoid ‘we’ and use a neutral phrasing as per the style guide?